src/exif.cpp

```diff
@@ -278,72 +278,36 @@ namespace Exiv2 {
 
     int Exifdatum::setValue(const std::string& value)
     {
-        if (value_.get() == nullptr) {
+        if (!value_) {
             TypeId type = key_->defaultTypeId();
             value_ = Value::create(type);
         }
         return value_->read(value);
     }
 
-    int Exifdatum::setDataArea(const byte* buf, size_t len)
-    {
-        return value_.get() == nullptr ? -1 : value_->setDataArea(buf, len);
-    }
+    int Exifdatum::setDataArea(const byte* buf, size_t len) { return value_ ? value_->setDataArea(buf, len) : -1; }
```

Comment:
I think this formatting style is not matching with the current clang-format settings we have in the project.

Anyways, it is not a big deal. We should probably pass clang-format over all the project soon and try to enforce keeping the format in the CI.

Reply:
That would be ideal, yes.